### PR TITLE
Added documentation for sense hat new flag (is_hat_attached)

### PR DESCRIPTION
**Pull request by nodinosaur**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<5883>

### CODE REVIEW

File: source/_components/sensor.sensehat.markdown [hunk 1]
```diff
@@ -34,5 +35,119 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
+#### Customizing the Sense HAT data
```
Comment (fabaff) [inline source/_components/sensor.sensehat.markdown]: You already added the details with #2009. Here only the new option should be included.

**Comment by nodinosaur** [comment]
Yes, I know, but I also wrote back that the page was completely broken. That a section was missing and the order was wrong.
This is how the page should be rendered.
You're choice if you want to embed this one.

File: source/_components/sensor.sensehat.markdown (hunk 1 above)
Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: Yes, I know, but I also wrote back that the page was completely broken. That a section was missing and the order was wrong.
This is how the page should be rendered.
You're choice if you want to embed this one.

Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: I should add, that that happened with the merger, not the way I wrote it. To correct it all, this version should be used. (_or let me fix / merge_)

Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: If you compare the original PR #2009 you will see that the text is different to what is on [/sensor.sensehat](https://home-assistant.io/components/sensor.sensehat/) So you will be able to judge for yourself that the merger failed somehow.
So this PR is essentially the same as #2009 with the addition of the new flag, and description.

**Pull request closed without merging**